common/workunit/workunit.cpp

```diff
@@ -3659,6 +3659,21 @@ class CLightweightWorkunitInfo : public CInterfaceOf<IConstWorkUnitInfo>
         appvalues.loadBranch(&p,"Application");
         totalThorTime = (unsigned)nanoToMilli(extractTimeCollatable(p.queryProp("@totalThorTime"), nullptr));
         _isProtected = p.getPropBool("@protected", false);
+
+        Owned<IPropertyTreeIterator> iterCostExecute = p.getElements("./Statistics/Statistic[@kind='CostExecute'][@scope='']");
```

Comment:
question: Could this more simply be coded as
`costExecute = p.getPropInt64("./Statistics/Statistic[@kind='CostExecute'][@scope='']/@value");
`
?